tests/bench/explicit.py

```
@@ -0,0 +1,1184 @@
+    # Try standard JSON format
+    try:
+        with open(path) as f:
+            data = json.load(f)
+
+        if isinstance(data, list):
+            return data
+        elif isinstance(data, dict):
+            # Single trace, wrap in list
+            return [data]
+        else:
+            raise ValueError(f"Unexpected JSON format in {path}")
```

Ruff (0.14.10)

Prefer `TypeError` exception for invalid type

(TRY004)